ts/utils/walletv2.ts

```diff
@@ -78,14 +79,24 @@ const getPaymentMethod = (
   return undefined;
 };
 
+// TODO: should be Either instead of return undefined
```

Comment:
```suggestion
// TODO: should be Option instead of return undefined
```
since we don't care about `Error` (left case) Option should be fit this case.
What do you think?

Reply:
I think in this case `Either` with an `Error` could  be also used to describe the failure of the attempt to transform a `PatchedWalletV2` into a `RawSatispayPaymentMethod`, instead to return an `undefined` (or `Option` `none`), something like a codec that fails!